File: packages/CVMTests/scripts/python/dataframe_int_column_type.py
#name: Python Int Column
#language: python
#output: dataframe resultInBound
#output: dataframe resultOutBound
resultInBound = pd.DataFrame.from_dict({'col1': [1000, 10000, 100000]})
resultOutBound = pd.DataFrame.from_dict({'col1': [1000, 10000, 2**31]})

File: packages/CVMTests/scripts/r/dataframe_int_column_type.R
#name: R Int Column
#language: r
#output: dataframe resultInBound
#output: dataframe resultOutBound


col1 <- as.integer(c(1000, 10000, 100000))
resultInBound <- data.frame(col1)
col1 <- c(1000, 10000, 2147483648)
resultOutBound <- data.frame(col1)